gluten-ut/spark33/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenCastSuite.scala

```diff
@@ -219,6 +219,32 @@ class GlutenCastSuite extends CastSuite with GlutenTestsTrait {
     checkEvaluation(cast(Literal.create(null, IntegerType), ShortType), null)
   }
 
+  test("cast Velox Timestamp to Int64 with floor division") {
```

Comment:
nit: please remove "Velox" from the test name. I assume the behavior on these test cases aligns with Spark and CH backend may support it finally.